staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics_test.go

```diff
@@ -65,15 +65,15 @@ func TestMetrics(t *testing.T) {
 			want: `
         # HELP authentication_attempts [ALPHA] Counter of authenticated attempts.
         # TYPE authentication_attempts counter
-        authentication_attempts{result="error"} 1
+        authentication_attempts{authenticator="unknown", result="error"} 1
```

Comment:
Can you update these tests to use a named authenticator?

Reply:
Done.